src/Microsoft.AspNetCore.Razor.Tools/ServerProtocol/ServerConnection.cs

```diff
@@ -23,12 +23,21 @@ internal static class ServerConnection
 
         public static bool WasServerMutexOpen(string mutexName)
         {
-            var open = Mutex.TryOpenExisting(mutexName, out var mutex);
-            if (open)
+            try
             {
-                mutex.Dispose();
-                return true;
+                var open = Mutex.TryOpenExisting(mutexName, out var mutex);
```

Comment:
Could we specifically try-catch opening the Mutex is that's what we care about? We don't trap exceptions thrown by `serverMutex.Dispose`